primitives/core/src/lib.rs

```diff
@@ -419,6 +419,14 @@ impl<T: Default> Get<T> for GetDefault {
 	}
 }
 
+/// A simple wrapper to turn a `Get<u16>` to `Get<u32>`.
+pub struct GetU16ToU32<T>(core::marker::PhantomData<T>);
+impl<T: Get<u16>> Get<u32> for GetU16ToU32<T> {
```

Comment:
Do you know why `MaxSignatories` is `u16` instead of `u32`?  
Since we have the MEL bound now, i think it can be changed.

Reply:
```
impl<G16: Get<u16>> Get<u32> for G16 {
  fn get() -> u32 { G16::get().into() }
}
```

and you won't need the struct.

Although I agree about keeping all the bounds u32 preferably.

Reply:
doesnt work. will just make it u32